### PR TITLE
Add the `cpu.vendor` hardware requirement for mrack

### PR DESCRIPTION
**Pull request by qcheng-redhat**
Pull Request Checklist

* [x] implement the feature
* [ ] write the documentation
* [x] extend the test coverage
* [ ] update the specification
* [ ] adjust plugin docstring
* [ ] modify the json schema
* [x] mention the version
* [x] include a release note

### CODE REVIEW

File: tmt/steps/provision/mrack.py [hunk 1]
```diff
@@ -275,6 +275,24 @@ def _transform_cpu_cores(
         children=[MrackHWBinOp('cores', beaker_operator, actual_value)])
 
 
+def _transform_cpu_vendor_name(
+        constraint: tmt.hardware.NumberConstraint,
```
Comment (therazix) [inline tmt/steps/provision/mrack.py:279]: Shouldn't it be `TextConstraint` instead of `NumberConstraint`?

Comment (qcheng-redhat) [reply tmt/steps/provision/mrack.py]: Hi, thanks for review!  Yes you are right. Updated.

**Review by therazix — APPROVED** [review]

**Review by thrix — APPROVED** [review]
LGTM

**Comment by happz** [comment]
@qcheng-redhat please, rebase the PR when you find some spare time.

**Comment by qcheng-redhat** [comment]
> @qcheng-redhat please, rebase the PR when you find some spare time.

Thanks for review, rebase is done.

**Comment by happz** [comment]
Unrelated failures, merging.

**Pull request merged**